Makefile

```diff
@@ -1,4 +1,6 @@
 .PHONY: test
 
 test:
+	docker-compose down
 	docker-compose run spec sh /app/test/run_tests.sh
+	docker-compose down
```

Comment:
make sure that any docker containers that are hanging around don't accidentally contaminate test runs.